modules/planning/conf/planner_open_space_config.pb.txt

```diff
@@ -32,6 +32,7 @@ dual_variable_warm_start_config {
     ipopt_recalc_y: "yes"
   }
   qp_format: OSQP
+  min_safety_distance: 0.0
```

Comment:
Theoretically this should be greater than 0.05 to avoid quantization error. Or we should specifically add a quantization error field so a lot of our parameters should be greater than it. Otherwise it might look good in planning/simulation but fail in vehicle. --- I will approve this as of now and let's take the discussion offline.

Reply:
> Theoretically this should be greater than 0.05 to avoid quantization error. Or we should specifically add a quantization error field so a lot of our parameters should be greater than it. Otherwise it might look good in planning/simulation but fail in vehicle. --- I will approve this as of now and let's take the discussion offline.

Sure, we will talk about this later.